### DISCUSSION TITLE
(question) Block hub with shields.io?

### DISCUSSION THREAD

**Discussion by ghost** [blockprotocol/blockprotocol#835] [Q&A] [answered]
Hi everyone.

I would like to know if you all already think about using shields in the block hub.

#### an idea
![image](https://user-images.githubusercontent.com/111701513/207531412-7e3e2f46-8f9a-45d6-bbcb-264e11511bd3.png)

**image-description:** As we can see we have a shields in block hub, everyone knows corresponding badges.

#### question/feedback
- Would it be possible to include metadata in the block components, informing used frameworks, programming language, if it has documentation or not? 
- Would it be possible to have shields in block hub?
- What do you all think of this idea?

**Comment by vilkinsons** [comment, marked as answer]
Rather than have a dedicated section for shields on block listing pages, I'd be in favour of ensuring we support the full gamut of [GitHub Flavored Markdown](https://github.github.com/gfm/) in READMEs.

This would effectively allow shields to be included while allowing READMEs in folk's GitHub repositories to be more faithfully reflected on the Þ Hub as a whole.

We might consider supporting multiple markdown specs (e.g. CommonMark [as used by Bitbucket](https://confluence.atlassian.com/bitbucketserver/markdown-syntax-guide-776639995.html), or [GitLab Flavored Markdown](https://docs.gitlab.com/ee/user/markdown.html) in the future - either at user's option or automatically (if able to auto-detect the source repo from GitHub/GitLab via a linked account).

**Reply by ghost** [reply]
Hi nonparibus.

> Rather than have a dedicated section for shields on block listing pages, I'd be in favour of ensuring we support the full gamut of [GitHub Flavored Markdown](https://github.github.com/gfm/) in READMEs.
> This would effectively allow shields to be included while allowing READMEs in folk's GitHub repositories to be more faithfully reflected on the Þ Hub as a whole.
> We might consider supporting multiple markdown specs (e.g. CommonMark [as used by Bitbucket](https://confluence.atlassian.com/bitbucketserver/markdown-syntax-guide-776639995.html), or [GitLab Flavored Markdown](https://docs.gitlab.com/ee/user/markdown.html) in the future - either at user's option or automatically (if able to auto-detect the source repo from GitHub/GitLab via a linked account).

thank you for feedback here!
